### PR TITLE
Various cleanups with regard to SegWit & block size

### PR DESCRIPTION
**Pull request by luke-jr**

### CODE REVIEW

File: _posts/en/posts/2016-01-26-segwit-benefits.md [hunk 1]
```diff
@@ -5,11 +5,24 @@ id: en-segwit-benefits
...
+* [Malleability Fixes][]
+* [Linear scaling of sighash operations][]
+* [Signing of input values][]
+* [Increased security for multisig via pay-to-script-hash (P2SH)][]
+* [Script versioning][]
+* [Reducing UTXO growth][]
+* [Compact fraud proofs][]
+* [Efficiency gains when not verifying signatures][]
+* [Block capacity/size increase][]
+* [Moving towards a single combined block limit][]
```
Comment (btcdrak) [inline _posts/en/posts/2016-01-26-segwit-benefits.md]: This is unnecessary, the TOC is rendered automatically. This change also doesnt render properly.

Comment (luke-jr) [inline _posts/en/posts/2016-01-26-segwit-benefits.md]: I don't see any clean overview on https://bitcoincore.org/en/2016/01/26/segwit-benefits/ ...

Comment (MarcoFalke) [inline _posts/en/posts/2016-01-26-segwit-benefits.md]: So what is wrong exactly?

Comment (luke-jr) [inline _posts/en/posts/2016-01-26-segwit-benefits.md]: It includes all the sub-sections, which scrolls the list off the page too fast.

**Comment by dcousens** [comment]
ACK

File: _posts/en/posts/2015-12-23-capacity-increases-faq.md [hunk 2]
```diff
@@ -16,8 +16,9 @@ New technology will be deployed when it is ready and has been tested. However, w
 | Feb 2016 | 0.12.0 | [libsecp256k1 verification][] |
 | Feb 2016 | | Segregated witness feature complete & ready for general review |
 | Mar 2016\* | 0.12.x | Deploy OP_CHECKSEQUENCEVERIFY (BIPs [68][BIP68] & [112][BIP112]) + [BIP113][] as first [BIP9][] versionbits soft fork |
-| April 2016\* |  0.12.x |  Deploy segregated witness |
-| 2016 | | Weak blocks, IBLTs, or both |
+| April 2016\* |  0.12.x |  Deploy segregated witness (including block size increase) |
+| 2016 | | Weak blocks and IBLT, Lightning, or both |
+| 2017 | | Prepare for hardfork |
```
Comment (btcdrak) [inline _posts/en/posts/2015-12-23-capacity-increases-faq.md]: This is technically out of date if we're going to be looking at potential HF proposals by July 2016

Comment (luke-jr) [inline _posts/en/posts/2015-12-23-capacity-increases-faq.md]: I don't know that we have consensus on moving this forward yet.

**Comment by luke-jr** [comment]
@btcdrak Please merge.

**Comment by jonasschnelli** [comment]
ACK

**Comment by btcdrak** [comment]
All merged except for 64ac7c9d04be94dbc8c19239c21595dd341917e1 and f2ddba8c42574c22ead45759c875008c9fb20e40

**Pull request closed without merging**